app/_src/guides/progressively-rolling-in-strict-mtls.md

```diff
@@ -149,19 +154,50 @@ Then we bring redis into the mesh by adding [kuma.io/sidecar-injection=true](/do
 
 ```bash
 kubectl patch deployment redis -n kuma-demo-migration \
---type='json' \
--p='[{"op": "add", "path": "/spec/template/metadata/labels/kuma.io~1sidecar-injection", "value": "enabled"}]'
+  --type='json' \
+  -p='[{"op": "add", "path": "/spec/template/metadata/labels/kuma.io~1sidecar-injection", "value": "enabled"}]'
 ```
 
 After this redis will be receiving plaintext traffic from non-meshed client.
-You can go to the {{site.mesh_product_name}} GUI, check the `Stats` tab for the `redis` Dataplane in the `kuma-demo-migration` namespace, and you should see this metric increment :
+You can check the `stats` for redis data plane:
 
-<center>
-<img src="/assets/images/guides/meshtls/dp-stats-view1.png" alt="Data Plane Proxies Stats metric for cluster.localhost_6379.upstream_cx_total"/>
-</center>
+```bash
+export REDIS_DPP_NAME=$(curl -s http://localhost:5681/meshes/default/dataplanes/_overview\?name\=redis | jq -r '.items[0].name')
+curl -s http://localhost:5681/meshes/default/dataplanes/$REDIS_DPP_NAME/stats | grep cluster.localhost_6379.upstream_cx
```

Comment:
why all metrics, not just `upstream_cx_total` as it was before?

Reply:
changed